tests/recipes/test_libgeos.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from unittest import mock
+from tests.recipes.recipe_lib_test import BaseTestForCmakeRecipe
+
+
+class TestLibgeosRecipe(BaseTestForCmakeRecipe, unittest.TestCase):
+    """
+    An unittest for recipe :mod:`~pythonforandroid.recipes.libgeos`
+    """
+    recipe_name = "libgeos"
+
+    @mock.patch("pythonforandroid.util.makedirs")
+    @mock.patch("pythonforandroid.util.chdir")
+    @mock.patch("pythonforandroid.build.ensure_dir")
+    @mock.patch("pythonforandroid.archs.glob")
+    @mock.patch("pythonforandroid.archs.find_executable")
+    def test_build_arch(
+        self,
+        mock_find_executable,
+        mock_glob,
+        mock_ensure_dir,
+        mock_current_directory,
+        mock_makedirs,
+    ):
+        # We overwrite the base test method because we
+        # want to avoid any file/directory creation
+        super(TestLibgeosRecipe, self).test_build_arch()
```

Comment:
I think we can actually ditch all theses Python2 style `super()` call and use the Python3 one if we wanted. At least in unit tests